docs/clients/sdk-by-examples/intro.md

```diff
@@ -0,0 +1,7 @@
+**SDK by Examples** offers an alternative and complementary way to learn about the Cosmos-SDK. It contains several examples that showcase how to build a decentralised application on top of the Cosmos-SDK from start to finish.  
```

Comment:
rename to readme? IMO it's better to have a `README.md` since that shows up nicely on Github, and a readme is basically an intro anyways